src/transaction/transaction_group_complete_manager.hpp

```diff
@@ -72,6 +72,9 @@ namespace cubtx
       void complete_logging (id_type group_id) override final;
 
     protected:
+      id_type set_current_group_minimum_transactions (unsigned int count_minimum_transactions,
```

Comment:
Let's summarize discussion postponed from previous patch:

Me and razvan argued it is better to move set_current_group_minimum_transactions/get_current_group_minimum_transactions from common virtual interface to salve GCM.

It is impossible to expect a minimum number of transactions in master or single GCM; it could block a smaller number of transactions for an unpredictable period of time.

To clearly understand the purpose of these functions, it is better to move it them to their usage context. Also, it seems like slave GCM expects an exact number of transactions rather than a minimum number.

Reply:
In single/master, db admin can set such parameter, to simulate actual group commit (see need_wakeup_LFT). Combined with a timer, this min transaction may be used in single/master. I expect the others opinion.

Reply:
I don't think we can/will have a minimum number of transactions in a group. We should also care for latency of each transaction as well as throughput.

Reply:
Now, that we know that it will be only a feature of slave GCM, we can move it in slave files.

However, moving is a risky job at this moment, since group mutex (stored in base class) is used to atomic check group state. Also, we don't want to break the actual design.

If we break the consistency, a hang may occur that will be quite difficult to catch.

So, my conclusion is that our focus should be consistency not perfect design, in this moment.

But, since some guys continuously insists, beyond my understanding, I tried to find a proper solution...